knowledge-base/how-to-change-report-server-iis-user.md

```diff
@@ -0,0 +1,70 @@
+---
+title: How to Create a limited IIS User for Report Server and Service Agent
+description: this tutorial shows how to use a lower privlidged user to host Report Server in IIS
+type: how-to
+page_title: How to change IIS user for Report Server and Service Agent
```

Comment:
The title leaves the impression that Service Agent uses something named 'IIS user', i.e. it works in IIS environment. I'd drop the 'IIS' part. 
Also, I believe the title could explain what will be the effect of the effort, e.g. 'what we'll achieve', rather than 'what we'll do right now'. In that sense, maybe 'How to run Report Server and Service Agent with limited permissions' would be better?

Reply:
It's a little late to change the URL and slug because it's linked to in multiple external resources... but I can use this better way to describe it as the page title, which wont affect navigation.